jetty-security/src/main/java/org/eclipse/jetty/security/authentication/SpnegoAuthenticator.java

```diff
@@ -72,8 +72,12 @@ public Authentication validateRequest(ServletRequest request, ServletResponse re
             return new DeferredAuthentication(this);
         }
 
+        final boolean hasNegotiateHeader = header == null ? false : header.startsWith(HttpHeader.NEGOTIATE.asString();
```

Comment:
I know this is the same stasrtsWith test as previous....  but headers are mean to be case insensitive, so might need a toLowercase in there.